src/app/course/practice-problem/practice-problem.component.html

```diff
@@ -2,7 +2,7 @@
 <table class="tui-table tui-space_left-4 tui-space_bottom-4">
     <tbody>
     <tr class="tui-table__tr">
-        <td class="tui-table__td">Your Success: {{userSuccessRate || 'N/A'}}</td>
+        <td class="tui-table__td">Tokens Earned: {{userSuccessRate || 'N/A'}}</td>
```

Comment:
How is tokens earned, userSuccessRate?

Reply:
Good point :). It was listed as an issue and I didn't notice should bring it up in the meeting.